README.md

```diff
@@ -50,3 +50,21 @@ Test your new function; I've provided some basic testing code in `code.test.js`.
 What is the worst-case asymptotic time complexity of your implementation? What
 is the worst-case asymptotic memory complexity? Add your answer, including your
 reasoning, to this markdown file.
+
+step 1, run though the matrix's length and create a starting list of cities. |v| runtime
+
+step 2, get the route's value and set it as current best route.
+|v| runtime as we only have 1 loop that runs though the route's length, which is the length of the adj matrix.
+
+step 3, make a new route and test it, if the value is less than the current best route then we havent found the minimum route yet so we need to make shure to continue searching.
+while loop runs untill we stop finding better solutions per iteration. each iteration has a complexity of |v|^3 because of the two nested for loops being |v| * |v|. Inside the inner forloop are two function calls that are each in the worst case |v|. so we get $|v| * |v| * (|v| + |v|) \in |v|^3$
+If there is a way to actuall find how many times the while loop runs it would be some number x (being the number of iterations required to collapse on a smallest number), this is atleast 10, but at most the (cbt = current best time) $\sum_{i=1}^{cbt} i + 1 
+
+The reason i had 10 was because it passed all the tests with 10 and i didnt change it. if my method of making new paths was actually conclusive I would have it do more... but its a really terrible method of trying to search different paths, it even re-does paths and re-tests paths that have allready been clasified as bad. so at one point i did try really big numbers to see if my results would improve... and they didnt so anything works with a emphasis on larger number roughly equals somewhat of better results.
```

Comment:
My point is that this is what actually determines the computational complexity, so you have to quantify this.